heufybot/modules/commands/log.py

```diff
@@ -39,8 +39,11 @@ def execute(self, server, source, command, params, data):
             logDate = date.today() - timedelta(delta)
         else:
             try:
-                formattedDate = params[0].split("-")
-                logDate = date(int(formattedDate[0]), int(formattedDate[1]), int(formattedDate[2]))
+                slashesAndDashes = ["/", "-"]
+                for symbol in slashesAndDashes:
+                    if symbol in params[0]:
+                        formattedDate = params[0].split(symbol)
+                        logDate = date(int(formattedDate[0]), int(formattedDate[1]), int(formattedDate[2]))
```

Comment:
I would recommend adding a `break` here to interrupt the for loop when either symbol has been found. Also, you want to make sure to check if `logdate is None` after your for loop and reply with a message if it is. Otherwise this will result in an error once it tries to convert `None` to a date.